server/controllers/postOrder.js

```diff
@@ -0,0 +1,35 @@
+const { insertOrder, insertOrderMeals } = require('../database/queries/order');
+const orderSchema = require('../validation/orderSchema');
+
+module.exports = (req, res, next) => {
+  let allMeals;
+  orderSchema
+    .validateAsync({
+      ...req.body, tableNumber: req.user.tableNumber,
+    })
+    .then((data) => {
+      const {
+        createdAt, totalPrice, meals,
+      } = data;
+      allMeals = meals;
+      return insertOrder(createdAt, totalPrice, req.user.tableNumber);
```

Comment:
Be consistent with what you use.
Use the table number in the data variable (which was validated).
```suggestion
      return insertOrder(createdAt, totalPrice, tableNumber); // data.tableNumber --> destructure it in line 12
```